<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="leUyeAd">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>55</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Üye Adı</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateVerilis">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Veriliş Tarihi</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Teslim Tarihi</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateTeslim">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btniptal">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>İPTAL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnKaydet">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>KAYDET</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leSecim">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Kitap Adı</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
